install.sh

```diff
@@ -133,7 +133,7 @@ else
     cat << EOS > "$rsystemd"/scylla-jmx.service.d/nonroot.conf
 [Service]
 EnvironmentFile=
-EnvironmentFile=$retc/sysconfig/scylla-jmx
+EnvironmentFile=$rsysconfdir/scylla-jmx
```

Comment:
Better to execute the translation here.

Reply:
I had moved the translation here. Thx.

The translation only cleaned a redundant backslash

```
# Without this fix:
EnvironmentFile=/home/scylla-test/scylladb/etc/sysconfig//scylla-jmx

# Applied this fix:
EnvironmentFile=/home/scylla-test/scylladb/etc/sysconfig/scylla-jmx
```